contracts/callpaths/WarmPath.sol

```diff
@@ -313,7 +308,7 @@ contract WarmPath is MarketSequencer, SettleLayer, ProtocolAccount {
         CurveMath.CurveState memory curve = snapCurve(poolKey);
         uint128 liq = Chaining.sizeConcLiq(qty, false, curve.priceRoot_,
                                            bidTick, askTick, inBase);
-        return burnConcentratedLiq(base, quote, poolIdx, bidTick, askTick,
+        return burnConcentratedLiq(base, quote, poolIdx,
```

Comment:
we should remove tick input from `burnConcentratedQty` similar to `mintConcentratedQty`